virtcontainers/acrn.go

```diff
@@ -444,6 +445,11 @@ func (a *Acrn) startSandbox(timeoutSecs int) error {
 		}
 	}()
 
+	if err := label.SetProcessLabel(a.config.ProcessLabel); err != nil {
```

Comment:
I wonder if ACRN needs selinux...
@vijaydhanraj  please take a look,

Reply:
Hi @devimc, I don't think selinux applies to acrn. Isn't this a linux kernel feature? or I am missing something? 
If the guest wants to enable or disable, we can pass it through the guest kernel parameters. But other than that not sure how it would apply for acrn hypervisor.

Reply:
Ok so arcn is not an executable that runs the VM?

Reply:
acrn-dm is the hypervisor device model that runs in the SOS (Service OS) and is responsible for creating guest UOS (user OS) VMs.